draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -336,12 +336,22 @@
    The RWS rule is used when at least one linear whitespace octet is required
    to separate field tokens. A sender &SHOULD; generate RWS as a single SP.
 </t>
+<t>
+   OWS and RWS have the same semantics as a single SP. Any content known to
+   be defined as OWS or RWS &MAY; be replaced with a single SP before
+   interpreting it or forwarding the message downstream.
+</t>
 <t>
    The BWS rule is used where the grammar allows optional whitespace only for
    historical reasons. A sender &MUST-NOT; generate BWS in messages.
    A recipient &MUST; parse for such bad whitespace and remove it before
    interpreting the protocol element.
 </t>
+<t>
+   BWS has the same semantics as an empty string. Any content known to be
```

Comment:
"BWS has the same semantics as an empty string" could be confused with an empty quoted-string.  How about "BWS has the same semantics as no whitespace", or more simply that "BWS has no semantics".